.env

```diff
@@ -0,0 +1 @@
+PORT = 8000
```

Comment:
- [ ] Kindly make sure the env file should be in `gitignore` file, as it may contain sensitive information of the project 😉😉💪

Reply:
Thanks @PowerLevel9000 for the catch. I have now added the file